src/runner.js

```diff
@@ -11,16 +11,25 @@ module.exports = async ({
   quarantineMode,
   debugMode,
   debugOnFail,
-  speed
+  speed,
+  concurrency,
+  app,
+  appInitDelay,
+  tags
 }) => {
   const testcafe = await createTestCafe(hostname, ...ports.slice(0, 2));
   const runner = testcafe.createRunner();
 
+  console.log("Running with browsers", browsers, "tags", tags, "concurrency", concurrency);
```

Comment:
I don't like this. The information seems to be redundant to me.

Reply:
Yep, I forgot to remove that